continuity/manager_relaystate.go

```diff
@@ -0,0 +1,151 @@
+package continuity
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/gofrs/uuid"
+	"github.com/pkg/errors"
+
+	"github.com/ory/herodot"
+	"github.com/ory/x/sqlcon"
+
+	"github.com/ory/kratos/session"
+	"github.com/ory/kratos/x"
+)
+
+var _ Manager = new(ManagerRelayState)
+var ErrNotResumableRelayState = *herodot.ErrBadRequest.WithError("no resumable session found").WithReasonf("The browser does not contain the necessary RelayState value to resume the session. This is a security violation and was blocked. Please try again!")
```

Comment:
No formatting needed in this error message

```suggestion
var ErrNotResumableRelayState = *herodot.ErrBadRequest.WithError("no resumable session found").WithReason("The browser does not contain the necessary RelayState value to resume the session. This is a security violation and was blocked. Please try again!")
```